src/ngraph/frontend/onnx_import/op/lstm.cpp

```diff
@@ -93,17 +94,20 @@ namespace ngraph
                             m_map[LSTMInput::LSTM_INPUT_W]->get_shape().front();
 
                         // ------ Optional inputs ------
-                        // The bias tensor for input gate. Shape [num_directions, 8*hidden_size]
+                        // The bias tensor for input gate. Shape [num_directions, 4*hidden_size]
                         if (ng_inputs.size() > 3 && !ng_inputs.at(3)->is_null())
                         {
-                            m_map[LSTMInput::LSTM_INPUT_B] = ng_inputs.at(3);
+                            auto bias = ng_inputs.at(3);
+                            auto splitted_bias = builder::split(bias, 2, 1);
```

Comment:
`splitted` is an nonstandard or archaic form of the past participle. Should be `split`
```suggestion
                            auto split_bias = builder::split(bias, 2, 1);
```